src/client/public/scripts/dash/dash.ts

```diff
@@ -324,15 +335,29 @@ function performPolling() {
 }
 
 async function start() {
-  window.addEventListener('focus', () => {
-    lastViewedData = lastPolledData;
+  const {data} = await getDashData();
+  renderDash(data, []);
 
+  window.addEventListener('focus', () => {
     // This will reset the favicon when the user revisits the page
     changeFavIcon(false);
+
+    // When an element is marked as 'is-newly-actionable' in the template
+    // it will also have the 'u-no-tranisitions' class applied. This means
+    // the browser will render the background with no transition.
+    // We then remove the u-no-transitions class and update the rendered
+    // template to transition out the flash
+    const elements = document.querySelectorAll('.is-newly-actionable');
+    for (let i = 0; i < elements.length; i++) {
+      const element = elements.item(i);
+      element.classList.remove('u-no-tranisitions');
+    }
```

Comment:
The more typical approach to this problem is to use CSS animations.

```
@keyframes flash {
  from {
     background-color: var(--actionable-highlight-color);
  }
  to {
    background-color: inherit;
  }
}
```

To ensure you can animate & restart the same animation, cause as style recalc:
```
element.classList.remove('flashClass'); // Ensure its not already animated
element.style.offsetTop; // Force a style recalc
element.classList.add('flashClass');
```

It's not too dissimilar from the approach here, but it'll reduce a little bit of the complexity: only one CSS class needed, only one point of logic.